db/schema.rb

```diff
@@ -572,11 +571,14 @@
     t.integer  "favourite_image_id"
     t.date     "stockit_moved_on"
     t.integer  "stockit_moved_by_id"
-    t.boolean  "saleable",                             default: false
+    t.boolean  "saleable",                 default: false
     t.integer  "set_item_id"
     t.string   "case_number"
     t.boolean  "allow_web_publish"
     t.integer  "received_quantity"
+    t.boolean  "last_allow_web_published"
```

Comment:
@namrataukirde do you know what that's about ?

Reply:
Correct @patrixr. I don't think it's used anywhere. 

It's not present in current live code: https://github.com/crossroads/api.goodcity/blob/live/db/schema.rb

And no new features or bug fixes uses this field. 

@abhinav098 Could you please help us to understand if it's used anywhere or it's added mistakenly?

Reply:
I am 100% sure it is not used anywhere. I think it's accidentally committed by someone. We can remove this field.